src/main/topLevelPages/myDashboard/newVirtualContributorWizard/useNewVirtualContributorWizard.tsx

```
@@ -405,23 +425,56 @@ const useNewVirtualContributorWizard = (): useNewVirtualContributorWizardProvide
+    }
 
-      setTryCreateCallout(false);
+    // create collection of docs & links
+    if (calloutData?.documents && calloutData?.documents.length > 0 && bokId) {
+      const documentsCallout = await handleCreateCallout(calloutDocumentsDetails);
+      const documentsCalloutId = documentsCallout?.id;
+      calloutId.current = documentsCalloutId;
+
+      // add documents to collection
+      if (documentsCalloutId) {
+        for (const document of calloutData?.documents) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)